pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -8795,18 +8792,16 @@ func createHelloWorldTaskRun(trName, ns, prName, pName string) *v1beta1.TaskRun
 	}
 }
 
-func createCancelledPipelineRun(prName string, specStatus v1beta1.PipelineRunSpecStatus) *v1beta1.PipelineRun {
-	return &v1beta1.PipelineRun{
-		ObjectMeta: baseObjectMeta(prName, "foo"),
-		Spec: v1beta1.PipelineRunSpec{
-			PipelineRef:        &v1beta1.PipelineRef{Name: "test-pipeline"},
-			ServiceAccountName: "test-sa",
-			Status:             specStatus,
-		},
-		Status: v1beta1.PipelineRunStatus{
-			PipelineRunStatusFields: v1beta1.PipelineRunStatusFields{
-				StartTime: &metav1.Time{Time: now},
-			},
-		},
-	}
+func createCancelledPipelineRun(t *testing.T, prName string, specStatus v1beta1.PipelineRunSpecStatus) *v1beta1.PipelineRun {
```

Comment:
Is this okay or should we be aiming to get rid of these helper functions?

Reply:
I would say yes 👼🏼